.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+    -   id: check-ast
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: requirements-txt-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/asottile/reorder_python_imports
+    rev: v3.8.2
+    hooks:
+    -   id: reorder-python-imports
+        args: [--py37-plus]
+-   repo: https://github.com/psf/black
+    rev: 22.8.0
+    hooks:
+    -   id: black
```

Comment:
Typically, you don't put black in a pre-commit because it gets annoying

Reply:
Why is that? I've used black in pre commit all the time

Reply:
not an issue